.github/workflows/release.yml

```diff
@@ -33,15 +33,25 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
+      - name: Login to Upbound
+        uses: docker/login-action@v1
+        with:
+          registry: xpkg.upbound.io
+          username: ${{ secrets.UPBOUND_MARKETPLACE_PUSH_ROBOT_USR }}
+          password: ${{ secrets.UPBOUND_MARKETPLACE_PUSH_ROBOT_PSW }}
+
+      - name: Push package image to upbound market
+        run: make upbound-package-push -e IMG_TAG=${GITHUB_REF##*/}
+
```

Comment:
Not needed, can be combined in the single make run below
```suggestion
```

Reply:
Will the push work on 2 different authorised repositories? Can you login in in multiple repos with `Docker login`?

Reply:
yes that works, we do this in a number of places.